### PR TITLE
Update HttpParser.swift

### PR DESCRIPTION
**Pull request by ghost**
Without the "!", Xcode 9 generates this error: "Value of optional type 'String?' not unwrapped; did you mean to use '!' or '?'?"

At this stage, it seems like **query** will have a value so force unwrapping it should be safe.

### CODE REVIEW

**Review by voynovia — COMMENTED** [review]
Force casts should be avoided.

File: Sources/HttpParser.swift [hunk 1]
```diff
@@ -40,7 +40,7 @@ public class HttpParser {
         guard url.endIndex > queryStart else {
             return []
         }
-        let query = String(url[queryStart..<url.endIndex])
+        let query = String(url[queryStart..<url.endIndex])!
```
Comment (voynovia) [inline Sources/HttpParser.swift]: Force cast is bad way

Comment (adamkaplan) [reply Sources/HttpParser.swift:43]: Here's a better solution https://github.com/httpswift/swifter/pull/321

**Comment by Vkt0r** [comment]
We can close this in favor of #321.

**Pull request closed without merging**